README.md

```diff
@@ -233,7 +292,13 @@ To integrate the [Okta Java SDK](https://github.com/okta/okta-sdk-java) into you
 </dependency>
 ```
 
-Then define the `okta.client.token` property. See [creating an API token](https://developer.okta.com/docs/api/getting_started/getting_a_token) for more info.
+Then define the following properties:
+
+| Key | Description |
+------|--------------
+| `okta.client.orgUrl` | Your Okta Url: `https://{yourOktaDomain}`, i.e. `https://dev-123456.okta.com` |
+| `okta.client.token` | An Okta API token, see [creating an API token](https://developer.okta.com/docs/api/getting_started/getting_a_token) for more info. |
+
```

Comment:
Looks like you need to merge `master` into `release_v2` ?

Reply:
Ref my comment above.